ui/components/app/assets/hooks/useMultichainAssets.tsx

```diff
@@ -0,0 +1,67 @@
+import { useSelector } from 'react-redux';
+import { Hex } from '@metamask/utils';
+import { getMultichainBalances } from '../../../../selectors/multichain';
+import {
+  getAccountAssets,
+  getAssetsMetadata,
+} from '../../../../selectors/assets';
+import { ParsedAssetId, parseAssetId } from '../util/parseAssetId';
+import { CHAIN_ID_TOKEN_IMAGE_MAP } from '../../../../../shared/constants/network';
+
+const accountId = 'e0bcfad0-2d3f-48cd-b50f-8a2f1d4f7dea';
```

Comment:
@salimtb what's the best way to get this from state? This account id doesn't match the public address for the solana account I've added in the snap.

Reply:
Okay, so same as evm. Cool, will update